src/components/Users/UserListAndCard.tsx

```
@@ -37,7 +37,11 @@
 
   return (
     <span
-      title={`${new Date(user.last_login).toLocaleString()}`}
+      title={
+        user.last_login
+          ? new Date(user.last_login).toLocaleString()
+          : undefined
```

ESLint

[error] Replace `⏎··········?·new·Date(user.last_login).toLocaleString()⏎·········` with `·?·new·Date(user.last_login).toLocaleString()`

(prettier/prettier)